automation/test.sh

```diff
@@ -198,11 +198,27 @@ set -e
 echo "Nodes are ready:"
 _oc get nodes
 
-_oc apply -f https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/kubevirt-operator.yaml
-_oc apply -f https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/kubevirt-cr.yaml
+_oc apply -f https://github.com/kubevirt/kubevirt/releases/download/${KV_VERSION}/kubevirt-operator.yaml
+_oc apply -f https://github.com/kubevirt/kubevirt/releases/download/${KV_VERSION}/kubevirt-cr.yaml
+
+# Deploy template validator (according to https://github.com/kubevirt/kubevirt-template-validator/blob/master/README.md)
+echo "Deploying template validator"
+rm -rf ../kubevirt-template-validator
+git clone https://github.com/kubevirt/kubevirt-template-validator ../kubevirt-template-validator
```

Comment:
I think it would be better to clone the same tag as `VALIDATOR_VERSION`, because the files in master may be different from what the released image expects.
`git clone -b ${VALIDATOR_VERSION} --depth 1`